Module_7.1/Q2.js

```diff
@@ -0,0 +1,18 @@
+let obj = {
+  a:"Apple",
+  b:["Basketball","Baseball"],
+  c: {
+   call: "cellphone"  },
+  d: "Dog"
+ }
+
+ function filterObject(obj) {
+  let result={};
+  for (let key in obj){
```

Comment:
You can also use, reduce function here which might be more suitable as we are accumulating the data